Add blog post for GodotConf 2023 Call for participations (#688)
godotengine/godot-website: collections/_article/godotcon-2023-cfp.md

---
title: "Godot Conference 2023: Call for Participations"
excerpt: "GodotCon 2023 needs participations"
categories: ["events"]
author: Julian
image: /storage/blog/covers/godotcon-2023.webp
image_caption_description: Image by Raffaele Picca
date: 2023-08-06 17:00:00
---

> We are organizing the annual Godot Conference that will happen [live in Munich](/article/godotcon-2023/) on November 4th and 5th 2023 for the 
> first time since [the last occurrence in 2020](/article/meet-community-fosdem-and-godotcon-2020/). We are really thrilled to meet again fellow Godot users and developers to share experiences during the two days of the event.

Reviewer: ```suggestion
We are organizing the annual Godot Conference that will happen [live in Munich](/article/godotcon-2023/) on November 4th and 5th 2023 for the 
first time since [our last conference in 2020](/article/meet-community-fosdem-and-godotcon-2020/). We are really thrilled to meet fellow Godot users and developers again and to share experiences during the two days of the event.
```

## Call for speakers & workshop

> We would like to propose all kinds of presentations during the event, and we need you for this! If you are interested in 
> sharing your project or experience that could interest Godot users, feel free to propose a dedicated talk or workshop on [our dedicated platform](https://events.godotengine.org/).

Reviewer: ```suggestion
We would like to include all kinds of presentations during the event, and we need you for this! If you are interested in 
sharing your project or experience that could interest Godot users, feel free to propose a dedicated talk or workshop on [our dedicated platform](https://events.godotengine.org/).
```

We are opening 2 tracks: 

* Main track
* Workshops

Please note that depending on the number of proposals we receive and the number of available slots, we may have to reject some proposals. Do not feel left out! You may propose again next year.

### Main track

This track is open to talks for various topics. We would like to see anything Godot-related or even not directly related to Godot but still relevant for game or engine developers.

> This track lasts the two whole days of the event. We propose two talk durations: **20 minutes or 50 minutes (+ 5-10 min Q&A)**, though if you need a different duration, you may express your needs in your proposal. When possible, we will change your talk duration accordingly.

Reviewer: These times are different from what is listed on events.godotengine.org. Does this article need to be updated or does events.godotengine.org need to be updated?

Author: events.godotengine.org proposes 30 minutes and 60 minutes talks. These correspond to maximum possible lengths 20+10 and 50+10.

Though we could change this article to `30 minutes or 60 minutes (**including** 5-10 min Q&A)`

Reviewer: Ah that makes sense. Either is fine. I'm happy to leave it as-is too

Reviewer: Ah that makes sense. Either is fine. I'm happy to leave it as-is too

Author: Changed nevertheless for the sake of clarity ;)

---
title: "Godot Conference 2023: Call for Participations"
excerpt: "GodotCon 2023 needs participations"
categories: ["events"]
author: Julian
image: /storage/blog/covers/godotcon-2023.webp
image_caption_description: Image by Raffaele Picca
date: 2023-08-06 17:00:00
---

> We are organizing the annual Godot Conference that will happen [lively at Munich](https://godotengine.org/article/godotcon-2023/) for the 

Reviewer 2: I assume this is what was meant!
```suggestion
We are organizing the annual Godot Conference that will happen [live in Munich](https://godotengine.org/article/godotcon-2023/) for the 
```

> first time since the last occurrence. We are really thrilled to meet again fellows Godot users and developers to share experiences during the 2 days of the event.

Reviewer 2: Numbers less than 10 should be written as words, in many newspaper etc. styleguides.

```suggestion
first time since the last occurrence. We are really thrilled to meet again fellows Godot users and developers to share experiences during the two days of the event.
```
Also, I don't know when the last occurrence was, but maybe good to put a year there, and link it to any existing old articles maybe.

> first time since the last occurrence. We are really thrilled to meet again fellows Godot users and developers to share experiences during the 2 days of the event.

Reviewer 3: ```suggestion
first time since the last occurrence. We are really thrilled to meet again fellow Godot users and developers to share experiences during the 2 days of the event.
```

## Call for speakers & workshop

We would like to propose all kinds of presentations during the event, and we need you for this! If you are interested in 
sharing your project or experience that could interest Godot users, feel free to propose a dedicated talk or workshop on [our dedicated platform](https://events.godotengine.org/).

We are opening 2 tracks: 

* Main track
* Workshops

> Please remind that depending on the number of proposals we will receive and the number of available slots, we may have to reject some proposals - do not feel left out! You may propose again next year.

Reviewer 2: ```suggestion
Please note that depending on the number of proposals we receive and the number of available slots, we may have to reject some proposals. Do not feel left out! You may propose again next year.
```

### Main track

This track is open to talks for various topics. We would like to see anything Godot-related or even not directly related to Godot but still relevant for game or engine developers.

> This track lasts the 2 whole days of the event. We propose 2 talk durations: **20 minutes or 50 minutes (+ 5-10 min Q&A)**, though if you need a different duration, you may express your needs in your proposal. When possible, we will change your talk duration accordingly.

Reviewer 2: ```suggestion
This track lasts the two whole days of the event. We propose two talk durations: **20 minutes or 50 minutes (+ 5-10 min Q&A)**, though if you need a different duration, you may express your needs in your proposal. When possible, we will change your talk duration accordingly.
```
OK, "5-10" there is an exception to the previously mentioned styleguide!

> Here are a few examples of last year's online GodotCon event:

Reviewer 2: ```suggestion
Here are a few examples from last year's online GodotCon event:
```

* WebXR in Godot
* (Almost) A Year of Fam Jams
* Teaching Godot Engine: Learning Experience Design
* Wwise Godot Integration: integrating audio middleware into Godot
* The Garden Path
* The new rendering of Godot 4.0
* Extending the 2D Renderer: Adding 3D-Like Shadowing

> * ...

Reviewer 2: ```suggestion
```

### Workshops

> This year we will provide a room specifically for workshops on both mornings of the event (on afternoons the room will be accessible for everyone to hack around). We may then dedicate from 2 to 4 slots (depending on their duration) for workshops. Ideally, a workshop may last from 1 to 2 hours, but if you think more time is needed for your proposal, please indicate how much longer it needs to be and why.

Reviewer 2: ```suggestion
This year we will provide a room specifically for workshops on both mornings of the event (on afternoons the room will be accessible for everyone to hack around). We may then dedicate from two to four slots (depending on their duration) for workshops. Ideally, a workshop may last from one to two hours, but if you think more time is needed for your proposal, please indicate how much longer it needs to be and why.
```

## Call for volunteers

A great event cannot happen without great staff to help us manage some tasks during the event:

* Steward Lead
* Stewards/General Volunteers
* Hosts (for speakers/sponsors)
* IT Lead
* IT Technicians
* Streaming Lead
* Streaming Operators & Moderators
* Ticketing & Accreditation Lead
* Ticketing & Accreditation
* Meet 2 Match Host

> But we are not limited to this role! If you think you can contribute, pelase do reach out!

Reviewer 2: ```suggestion
But we are not limited to these roles! If you think you can contribute, please do reach out.
```

> We understand you may also be interested in participating the event yourself as well. Therefore, volunteer duties will not last all the event long - unless you expressively tell us so.

Reviewer 2: ```suggestion
We understand volunteers may be interested in participating in the event as well. Therefore, volunteer duties will not last all event long - unless you expressively tell us so.
```

If you are interested in one of the above roles, please send us an email at [godotcon2023\@godotengine.org](mailto:godotcon2023@godotengine.org?subject=Volunteering Inquiry).

# Sponsoring 

To enable an event for about 300 participants and to keep ticket prices low, we have to gather some external funds, mainly for the venue, security, catering, documentation, swag, speaker support, etc. Therefore we are still looking for new sponsors to help us make it happen.

As a sponsor you have a distinctive chance to support this conference, which offers a remarkable developer relations and marketing opportunity. This opportunity comes with the added advantage of relatively low investment. If you wish to sponsor this event, please send us an email at [godotcon2023\@godotengine.org](mailto:godotcon2023@godotengine.org?subject=Sponsorship Inquiry) and provide some detail on how or what you would like to sponsor. For more informations, please have a look at our [Sponsorship Page](https://galvanized-slope-59b.notion.site/Public-GodotCon-Sponsorship-Page-acd1185de6594982943682caf4840d67).

> We would like to address a big thank you to our current sponsors without whom this event would not be possible.

Reviewer 2: ```suggestion
We would like to address a big thank you to our current sponsors without whom this event would not be possible:
```
